layers/+lang/c-c++/funcs.el

```diff
@@ -53,6 +54,21 @@
                     (split-string (buffer-string) "\n" t))))
     compile-flags))
 
+(defun spacemacs//c-c++-get-standard-include-paths (lang)
+  (let* ((start "#include <...> search starts here:")
+         (end "End of search list.")
+         (gcc-tmplt "echo | gcc -x%s -E -v - 2>&1")
```

Comment:
Looking into the GCC source, the lines that print the include directories were last changed in January 2003. GCC 3.2.2 was released February 2003. I don't know if that is sufficient to you. I was thinking that perhaps some sort of check should be done for the existence of GCC and sed on the system, and then make sure that the extracted lines really are absolute search paths. If one of these checks fail, a default of /usr[/local]/include could be returned.